mysite/templates/mysite/base_generic.html

```diff
@@ -0,0 +1,92 @@
+<html>
+<head>
+
+    {% block head-css %}
+        <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/bootstrap/3.2.0/css/bootstrap.min.css">
+        <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/bootstrap/3.2.0/css/bootstrap-theme.min.css">
+        <style>
+            html {
+                position: relative;
+                min-height: 100%;
+            }
+
+            body {
+                margin-bottom: 60px;
+            }
+
+            footer {
+                position: absolute;
+                left: 0;
+                right: 0;
+                bottom: 0;
+                background-color: black;
+                padding: 15px 15px 0;
+                color: white;
+            }
+        </style>
```

Comment:
Put this style into some separate file and include that file here.

Reply:
Feedback Incorporated.